src/monitors/issue.mjs

```diff
@@ -64,7 +64,7 @@ class Issue extends Monitor {
 		await msg.react('🗑');
 
 		try {
-			await msg.awaitReactions((reaction, user) => reaction.emoji.name === '🗑' && !user.bot, {
+			await msg.awaitReactions((reaction, user) => reaction.emoji.name === '🗑' && !user.bot && user.id === msg.author.id, {
```

Comment:
A bot can't trigger this in the first place, I suggest removing the `!user.bot` check.